plugins/toolbox/src/components/Generators/QR.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { DefaultEditor } from '../DefaultEditor';
+import { faker } from '@faker-js/faker';
+import QRCode from "react-qr-code";
+
+export const QR = () => {
```

Comment:
Could be named `QRCode` but then we have to name the `import QRCode from "react-qr-code";` differently. Would be more aligned with the changes proposed in `tools.tsx`.